src/AppBundle/DataFixtures/ORM/PostFixtures.php

```diff
@@ -107,8 +106,6 @@ private function addRandomTags(Post $post)
         }
     }
 
-=======
->>>>>>> pull/431:src/AppBundle/DataFixtures/ORM/LoadFixtures.php
```

Comment:
btw, this conflict could be resolved in standalone PR ;)
to keep PRs atomic and clean